data/offers/test.js

```diff
@@ -0,0 +1,145 @@
+const {performance, PerformanceObserver} = require('perf_hooks');
```

Comment:
javascript?

Reply:
What is this even?

Reply:
I perfed what is the best way to merge two lists of objects, accidentally added the file.